README.md

```diff
@@ -3,3 +3,7 @@
 [BcnEng](http://bcneng.net) is a slack community built around the software engineering community from Barcelona.
 
 You can get an invitation to join it [here](http://slack.bcneng.net).
+
+Also please read our:
+- [Code of Conduct](https://github.com/bcneng/bcneng/blob/master/coc/README.md)
+- Netiquette (WIP)
```

Comment:
Add the link already. Leave the `WIP` so it's known to be prone to changes though.